### PR TITLE
Fixes Explorer shuttle & Makes the dallus into an operational ship

### PR DESCRIPTION
**Pull request by GremlingSS**
Shuttle functions and lands! Woo!
Everything is completed though the shuttle could do with a rework and being slightly shorter, it doesn't fit into most landing spots, maybe move the door up to the left too so it creates a air-tight lock :)

### CODE REVIEW

**Comment by GremlingSS** [comment]
oops i used the wrong branch as the base so it has the other PR attached to this too

**Comment by GremlingSS** [comment]
ooh didn't notice this error, mb! Will clean it up now

**Comment by TalkingCactus** [comment]
What's holding this back?

**Comment by GremlingSS** [comment]
Z Level 17 for some reason forces itself to be 255x255 even though it's smaller. So if the map is loaded in, it will generate space within the rest of the area. I noticed it's always Z level 17 no matter what uses it, and if I remove Z level 17, it will still generate stuff there with empty space so I kinda had given up trying to figure out why it's doing it.

I noticed it's always space, there's no area, though I haven't checked if the Z level is labelled something else now that I've removed the beach planet from there in my own testing.

So to summarise:
Z level 17 is forced to be 255x255, even though the map is smaller, and I can't just make the map bigger.

**Comment by TalkingCactus** [comment]
which map file is z17?
